### ISSUE TITLE
Is it possible to add our own tool buttons?

### ISSUE THREAD

**Issue by mark8044** [glynskyi/zefyrka#23]
Is it possible to add our own tool buttons?  Love the editor so far, but the toolbar is very basic...

**Comment by nipunasudha** [comment]
Yes you can create your own toolbar instance and add any existing button to it. Also, you can create your own buttons, take a look at the built-in button definitions to get an idea of how to make them.
